src/test/resources/environments.json

```diff
@@ -1,8 +1,14 @@
 {
   "prod": {
     "BASE_URL_API": "http://the-internet.herokuapp.com",
-    "BASE_URL": "http://the-internet.herokuapp.com/",
+    "BASE_URL1": "http://the-internet.herokuapp.com/",
```

Comment:
what is BASE_URL1? It has to be a meaningful name to allow easy usage at later point in time